repositories.bzl

```diff
@@ -16,8 +16,9 @@
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_jar")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
 
-def jazzer_dependencies():
+def jazzer_dependencies(is_android = False):
```

Comment:
`is_android` reads as if we are fetching these deps when building *on* instead of *for* Android. Maybe just call the arg `android`?